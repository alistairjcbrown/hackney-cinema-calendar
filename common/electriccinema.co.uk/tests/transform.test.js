const moviesWithLocalScreenings = require("./mock-data/movie-with-local-screenings.json");
const transform = require("../transform");

jest.useFakeTimers().setSystemTime(new Date("2024-10-22"));

describe("Electric Cinema (common)", () => {
  describe("when no movies provided", () => {
    it("returns an empty list of events", async () => {
      const value = {
        filmData: { films: {}, screenings: {}, screeningTypes: {} },
      };
      expect(
        await transform(
          { cinemaId: "7497", domain: "https://www.electriccinema.co.uk" },
          value,
          {},
        ),
      ).toEqual([]);
    });
  });

  describe("when movies with local screenings provided for Portobello", () => {
    it("returns a list of events", async () => {
      const value = moviesWithLocalScreenings;
      expect(
        await transform(
          { cinemaId: "7497", domain: "https://www.electriccinema.co.uk" },
          value,
          {},
        ),
      ).toMatchSnapshot();
    });
  });

  describe("when movies with local screenings provided for White City", () => {
    it("returns a list of events", async () => {
      const value = moviesWithLocalScreenings;
      expect(
        await transform(
          { cinemaId: "55140", domain: "https://www.electriccinema.co.uk" },
          value,
          {},
        ),
      ).toMatchSnapshot();
    });
  });
});
