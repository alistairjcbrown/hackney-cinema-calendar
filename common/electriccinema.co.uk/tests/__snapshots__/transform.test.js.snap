// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Electric Cinema (common) when movies with local screenings provided for Portobello returns a list of events 1`] = `
[
  {
    "overview": {
      "actors": [],
      "age-restriction": "PG",
      "categories": [],
      "directors": [
        "Tim Burton",
      ],
      "duration": 5400000,
      "year": "2012",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755671",
        "notes": "Kids Club",
        "screen": "1",
        "time": 1729929600000,
      },
    ],
    "title": "Frankenweenie",
    "url": "https://www.electriccinema.co.uk/film/frankenweenie/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Oz Perkins",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756718",
        "notes": "",
        "screen": "1",
        "time": 1730408400000,
      },
    ],
    "title": "Longlegs (Halloween Re-release)",
    "url": "https://www.electriccinema.co.uk/film/longlegs/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Ellen Kuras",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755668",
        "notes": "",
        "screen": "1",
        "time": 1729615500000,
      },
    ],
    "title": "Lee",
    "url": "https://www.electriccinema.co.uk/film/lee/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Nora Fingscheidt",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755669",
        "notes": "",
        "screen": "1",
        "time": 1729701900000,
      },
    ],
    "title": "The Outrun",
    "url": "https://www.electriccinema.co.uk/film/the-outrun/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Eduardo Sanchez • Daniel Myrick",
      ],
      "duration": 5400000,
      "year": "1999",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/754301",
        "notes": "Special Event",
        "screen": "1",
        "time": 1729981800000,
      },
    ],
    "title": "1999 Season: The Blair Witch Project",
    "url": "https://www.electriccinema.co.uk/film/the-blair-witch-project/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Edgar Wright",
      ],
      "duration": 5400000,
      "year": "2004",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/754300",
        "notes": "Special Event",
        "screen": "1",
        "time": 1729971900000,
      },
    ],
    "title": "Shaun of the Dead (20th Anniversary)",
    "url": "https://www.electriccinema.co.uk/film/shaun-of-the-dead-20th-anniversary/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Ali Abbasi",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755685",
        "notes": "Electric Subtitled",
        "screen": "1",
        "time": 1729626300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755686",
        "notes": "",
        "screen": "1",
        "time": 1729712700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755687",
        "notes": "",
        "screen": "1",
        "time": 1729785600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756610",
        "notes": "",
        "screen": "1",
        "time": 1729875600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756611",
        "notes": "",
        "screen": "1",
        "time": 1729950300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756612",
        "notes": "",
        "screen": "1",
        "time": 1730034000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756613",
        "notes": "",
        "screen": "1",
        "time": 1730148300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/the-apprentice/",
        "notes": "Electric Subtitled",
        "screen": "1",
        "time": 1730223900000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756636",
        "notes": "",
        "screen": "1",
        "time": 1730321100000,
      },
    ],
    "title": "The Apprentice",
    "url": "https://www.electriccinema.co.uk/film/the-apprentice/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Ridley Scott",
      ],
      "duration": 5400000,
      "year": "2000",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755670",
        "notes": "",
        "screen": "1",
        "time": 1729797300000,
      },
    ],
    "title": "Gladiator",
    "url": "https://www.electriccinema.co.uk/film/gladiator-2000/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "18",
      "categories": [],
      "directors": [
        "Sean Baker",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756701",
        "notes": "Special Event",
        "screen": "1",
        "time": 1730045700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756719",
        "notes": "",
        "screen": "1",
        "time": 1730493000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756720",
        "notes": "",
        "screen": "1",
        "time": 1730558700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756721",
        "notes": "",
        "screen": "1",
        "time": 1730639700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/anora/",
        "notes": "Electric Scream!",
        "screen": "1",
        "time": 1730728800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756723",
        "notes": "",
        "screen": "1",
        "time": 1730741400000,
      },
    ],
    "title": "Anora",
    "url": "https://www.electriccinema.co.uk/film/anora/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "12A",
      "categories": [],
      "directors": [
        "Pedro Almodóvar",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756698",
        "notes": "",
        "screen": "1",
        "time": 1729886400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756699",
        "notes": "",
        "screen": "1",
        "time": 1729939500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756700",
        "notes": "",
        "screen": "1",
        "time": 1729961100000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756712",
        "notes": "",
        "screen": "1",
        "time": 1730058300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756714",
        "notes": "Electric Scream!",
        "screen": "1",
        "time": 1730125800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756713",
        "notes": "",
        "screen": "1",
        "time": 1730137500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756715",
        "notes": "",
        "screen": "1",
        "time": 1730234700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756716",
        "notes": "",
        "screen": "1",
        "time": 1730310300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/the-room-next-door/",
        "notes": "",
        "screen": "1",
        "time": 1730396700000,
      },
    ],
    "title": "The Room Next Door",
    "url": "https://www.electriccinema.co.uk/film/the-room-next-door/",
  },
  {
    "overview": {
      "actors": [],
      "categories": [],
      "directors": [
        "Dougal Wilson",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731078000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731163500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731174300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731245400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731256200000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "Electric Scream!",
        "screen": "1",
        "time": 1731333600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731347100000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "Electric Subtitled",
        "screen": "1",
        "time": 1731433500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731519900000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731606300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731769200000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731779400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
        "notes": "",
        "screen": "1",
        "time": 1731861900000,
      },
    ],
    "title": "Paddington in Peru",
    "url": "https://www.electriccinema.co.uk/film/paddington-in-peru/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Ridley Scott",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/gladiator-ii/",
        "notes": "",
        "screen": "1",
        "time": 1731702600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/gladiator-ii/",
        "notes": "",
        "screen": "1",
        "time": 1731789900000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/gladiator-ii/",
        "notes": "",
        "screen": "1",
        "time": 1731849300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/gladiator-ii/",
        "notes": "",
        "screen": "1",
        "time": 1731872700000,
      },
    ],
    "title": "Gladiator II",
    "url": "https://www.electriccinema.co.uk/film/gladiator-ii/",
  },
]
`;

exports[`Electric Cinema (common) when movies with local screenings provided for White City returns a list of events 1`] = `
[
  {
    "overview": {
      "actors": [],
      "age-restriction": "PG",
      "categories": [],
      "directors": [
        "Tim Burton",
      ],
      "duration": 5400000,
      "year": "2012",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756731",
        "notes": "Kids Club",
        "screen": "1",
        "time": 1730026800000,
      },
    ],
    "title": "Frankenweenie",
    "url": "https://www.electriccinema.co.uk/film/frankenweenie/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "18",
      "categories": [],
      "directors": [
        "Coralie Fargeat",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755738",
        "notes": "",
        "screen": "3",
        "time": 1729624200000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755742",
        "notes": "",
        "screen": "3",
        "time": 1729710600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755746",
        "notes": "",
        "screen": "3",
        "time": 1729797000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756659",
        "notes": "",
        "screen": "3",
        "time": 1730232000000,
      },
    ],
    "title": "The Substance",
    "url": "https://www.electriccinema.co.uk/film/the-substance/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Eduardo Sanchez • Daniel Myrick",
      ],
      "duration": 5400000,
      "year": "1999",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756679",
        "notes": "Special Event",
        "screen": "2",
        "time": 1730408400000,
      },
    ],
    "title": "1999 Season: The Blair Witch Project",
    "url": "https://www.electriccinema.co.uk/film/the-blair-witch-project/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Ali Abbasi",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755735",
        "notes": "Electric Scream!",
        "screen": "1",
        "time": 1729596600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755736",
        "notes": "Electric Subtitled",
        "screen": "2",
        "time": 1729602000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756210",
        "notes": "",
        "screen": "1",
        "time": 1729615500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755739",
        "notes": "",
        "screen": "1",
        "time": 1729701900000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755743",
        "notes": "",
        "screen": "1",
        "time": 1729788300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756637",
        "notes": "",
        "screen": "3",
        "time": 1729882800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756639",
        "notes": "",
        "screen": "1",
        "time": 1729954800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756649",
        "notes": "",
        "screen": "3",
        "time": 1730055600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756652",
        "notes": "",
        "screen": "3",
        "time": 1730146500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756676",
        "notes": "",
        "screen": "3",
        "time": 1730320200000,
      },
    ],
    "title": "The Apprentice",
    "url": "https://www.electriccinema.co.uk/film/the-apprentice/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "U",
      "categories": [],
      "directors": [
        "Chris Sanders",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755737",
        "notes": "",
        "screen": "3",
        "time": 1729614600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755741",
        "notes": "Electric Subtitled",
        "screen": "3",
        "time": 1729701000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755745",
        "notes": "",
        "screen": "3",
        "time": 1729787400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756638",
        "notes": "",
        "screen": "3",
        "time": 1729873800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756640",
        "notes": "",
        "screen": "2",
        "time": 1729940400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756647",
        "notes": "",
        "screen": "3",
        "time": 1730035800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756648",
        "notes": "",
        "screen": "3",
        "time": 1730045700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756651",
        "notes": "",
        "screen": "3",
        "time": 1730136600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756655",
        "notes": "",
        "screen": "3",
        "time": 1730214000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756656",
        "notes": "",
        "screen": "3",
        "time": 1730223000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756678",
        "notes": "",
        "screen": "3",
        "time": 1730395800000,
      },
    ],
    "title": "The Wild Robot",
    "url": "https://www.electriccinema.co.uk/film/the-wild-robot/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "18",
      "categories": [],
      "directors": [
        "Parker Finn",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756211",
        "notes": "",
        "screen": "1",
        "time": 1729626300000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755740",
        "notes": "",
        "screen": "1",
        "time": 1729712700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755744",
        "notes": "",
        "screen": "1",
        "time": 1729799100000,
      },
    ],
    "title": "Smile 2",
    "url": "https://www.electriccinema.co.uk/film/smile-2/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Kelly Marcel",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755747",
        "notes": "",
        "screen": "1",
        "time": 1729875600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/755748",
        "notes": "",
        "screen": "1",
        "time": 1729885500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756641",
        "notes": "",
        "screen": "3",
        "time": 1729936800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756642",
        "notes": "",
        "screen": "3",
        "time": 1729946700000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756643",
        "notes": "",
        "screen": "3",
        "time": 1729956000000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756644",
        "notes": "",
        "screen": "1",
        "time": 1730037600000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756645",
        "notes": "",
        "screen": "1",
        "time": 1730047500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756646",
        "notes": "",
        "screen": "1",
        "time": 1730057400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756650",
        "notes": "",
        "screen": "1",
        "time": 1730137500000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/venom-the-last-dance/",
        "notes": "Electric Scream!",
        "screen": "1",
        "time": 1730206800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/venom-the-last-dance/",
        "notes": "Electric Subtitled",
        "screen": "2",
        "time": 1730210400000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756653",
        "notes": "",
        "screen": "1",
        "time": 1730223900000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756654",
        "notes": "",
        "screen": "1",
        "time": 1730233800000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/film/venom-the-last-dance/",
        "notes": "Electric Subtitled",
        "screen": "1",
        "time": 1730311200000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756674",
        "notes": "",
        "screen": "1",
        "time": 1730321100000,
      },
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756677",
        "notes": "",
        "screen": "1",
        "time": 1730407500000,
      },
    ],
    "title": "Venom: The Last Dance",
    "url": "https://www.electriccinema.co.uk/film/venom-the-last-dance/",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "15",
      "categories": [],
      "directors": [
        "Scott Beck • Bryan Woods",
      ],
      "duration": 5400000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://www.electriccinema.co.uk/tickets/756680",
        "notes": "",
        "screen": "3",
        "time": 1730405700000,
      },
    ],
    "title": "Heretic",
    "url": "https://www.electriccinema.co.uk/film/heretic/",
  },
]
`;
